test/expected.css

```diff
@@ -10034,4 +10035,4 @@ a.text-dark:hover, a.text-dark:focus {
     border-color: #dee2e6;
     color: inherit;
   }
-}
\ No newline at end of file
+}
```

Comment:
I honestly don't know where this comes from... I'm pretty sure there's no end of line in my copy either :/